multicluster/controllers/multicluster/member/gateway_controller.go

```diff
@@ -132,6 +127,7 @@ func (r *GatewayReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		}
 		return ctrl.Result{}, nil
 	}
+	r.serviceCIDR = gw.ServiceCIDR
```

Comment:
You can remove the serviceCIDR in NewGatewayReconciler if you will get it from Gateway.

Reply:
Yes, and we should remove `serviceCIDR` from the GatewayReconciler struct.